sdk/nodejs/tsconfig.json

```diff
@@ -17,7 +17,6 @@
         "billingCycle.ts",
         "config/index.ts",
         "config/vars.ts",
-        "connect.ts",
```

Comment:
Do we need to delete the files? Is that a breaking change?

Reply:
This was autogenerated... I am sure this is related to https://github.com/terraform-providers/terraform-provider-packet/blob/master/CHANGELOG.md#270-november-19-2019

Reply:
Why are `connect.ts` and friends still there?

Reply:
The last release we made of the provider was 2.6.1 when the old resource was deprecated

Reply:
also, it doesn't seem that our generation cleans out old files as part of the SDK generation

Reply:
So what's the normal action then? Delete it manually? Or let them pile on ~forever?